### PR TITLE
Fix Sts Cyril and Methodius celebration in SK and CZ calendars

### LINKED ISSUE #117: `slovakia`: fix the celebration of `saintsCyrilMonkAndMethodiusBishop`
Related #110

---

`saintsCyrilMonkAndMethodiusBishop` issue is somewhat fixed, but there are still some (sub)issues to be fixed:
- [x] define the celebration on fixed date of 5 July as that celebration is _always_ celebrated;
- [x] change it’s celebration rank to `SOLEMNITY` both in both `slovakia` and `czechRepublic` calendars (I have forgotten to check this);
- [ ] ~create a votive Mass on 14 Feb~ (this will be fixed in #110);
- [x] remove the boolean flag `saintsCyrilMonkAndMethodiusBishopOnFeb14` as it is useless (@emagnier this task I assign to you :smiley:):
  - [x] note that this should be removed from both `slovakia` and `czechRepublic` calendars (or any other);
  - [x] remove the `04_calendar_date_overrides.js` test.

### PR DESCRIPTION
**Pull request by tukusejssirs**
- Remove `saintsCyrilMonkAndMethodiusBishopOnFeb14` option (fixes #117)

### CODE REVIEW

**Comment by tukusejssirs** [comment]
I see the Travis check fails which is sort of expected as there is `test/03_calendars.js` test, which should be removed altogether. Can I simply delete this file or is there anything else that needs to be done?

**Comment by tukusejssirs** [comment]
@emagnier, will you do that or will you instruct me what do I need to remove? :smiley:

**Comment by emagnier** [comment]
You can search all occurrences of `saintsCyrilMonkAndMethodiusBishopOnFeb14` in the codebase and the documentation, and remove them. Let me know if you prefer me to do it.

**Comment by tukusejssirs** [comment]
@emagnier, should be done.

File: src/calendars/czechRepublic.js [hunk 1]
```diff
@@ -110,10 +109,8 @@ let dates = (year, saintsCyrilMonkAndMethodiusBishopOnFeb14 = false ) => {
     // https://en.wikipedia.org/wiki/Saints_Cyril_and_Methodius
     {
       "key": "saintsCyrilMonkAndMethodiusBishop",
-      "type": Types.FEAST,
-      "moment": ((y, flag) => {
-        return flag ? moment.utc({ year: year, month: 1, day: 14 }): moment.utc({ year: year, month: 6, day: 5 });
-      })(year, saintsCyrilMonkAndMethodiusBishopOnFeb14),
+      "type": Types.SOLEMNITY,
```
[the remark this answers is not in the document]
Comment (emagnier) [reply src/calendars/czechRepublic.js]: Ok, moving forward on it to prepare the 1.3.0 release.

[the remark this answers is not in the document]
Comment (tukusejssirs) [reply src/calendars/czechRepublic.js]: > You changed it to a solemnity, is that correct?

Yes, I noted this in #117. I forgot to fix it when I first talked about this with @pejulian and also when adding and fixing the Slovak l10n and feasts last month. As Sts Cyril and Methodius came to Slovakia / Czech Republic (the former Great Moravia, which spread across SK/CZ common borders), these are quite important saints for us. :smiley:

**Comment by tukusejssirs** [comment]
Anyway, @emagnier, when merging to PRs, could you edit the merge messages? IMO, they should include the `fixes #nnn` special words. Currently, we need to check manually (again) which issues we must manually close.

Or do you propose something else? :smiley:

File: src/calendars/czechRepublic.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (emagnier) [reply src/calendars/czechRepublic.js]: Yes, for all slav people 😃 
Thank you for your comment.

[the remark this answers is not in the document]
Comment (tukusejssirs) [reply src/calendars/czechRepublic.js]: > Yes, for all slav people smiley

Although in certain sense you might be right, but IMO/AFAIK they _solemnly_ celebrated in CZ/SK and in Thessaloniki, Greece (as they come from [around] there). :smiley:

**Comment by emagnier** [comment]
Here it's already in the description of the commit: https://github.com/romcal/romcal/commit/e8ea215cce912c613920f5233b2bafac29c86d91

**Comment by tukusejssirs** [comment]
Indeed. This might be my fault as I have not edited the description after my final changes and it does not reflect everything I did. As for the `fix` issue, there is the word `partially`, which I added there on purpose for when I created this PR, I was not determined to fix #117 in whole. In order to auto-fix an issue, there must be no word between the special word (e.g. `fix`) and the issue reference(s) (e.g. `#117`), except for other issues and space separators (e.g. this is valid: `fixes #1 #2 #3`).

Do we want to amend the merge message?

**Comment by emagnier** [comment]
Done https://github.com/romcal/romcal/commit/3ebe0a97549d1035044c2067c350716db4d258a1.

**Pull request merged**